bears/r/FormatRBear.py

```diff
@@ -20,12 +20,12 @@ class FormatRBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         r_keep_comments: bool=True,
+                         r_keep_comments: bool=None,
```

Comment:
in this case I would keep the default, a certain default config is useful for beginners.
I only wanted you to disable the max-expression-length as it is too advanced to be checked by default imo :)